### DISCUSSION TITLE
Trying to use nextbox-ui-plugin on NetBox but facing error : Must be "SavedTopology" instance.

### DISCUSSION THREAD

**Discussion by NaveenReddy198** [netbox-community/netbox#14651] [Help Wanted!]
Hii Team , Iam using Netbox and on top of that i wanted to use nextbox-ui-plugin .I have installed it and configured it .
While running below command 
(venv) $ python3 manage.py migrate
Iam getting below errors 
ERRORS:
nextbox_ui_plugin.SavedTopology.created_by: (fields.E300) Field defines a relation with model 'users.AdminUser', which is either not installed, or is abstract.
nextbox_ui_plugin.SavedTopology.created_by: (fields.E307) The field nextbox_ui_plugin.SavedTopology.created_by was declared with a lazy reference to 'users.adminuser', but app 'users' doesn't provide model 'adminuser'.
Can anyone help me with this ?

**Comment by candlerb** [comment]
Since this is a bug in the plugin, you're best off asking on the plugin's mailing list or issue tracker. You'll need to say, at minimum, what version of Netbox you're using. Netbox's API changes between versions, and plugins need to be modified to work with newer versions.

**Comment by rfj2024** [comment]
Boa noite,

Estou com mesmo problema.

https://www.piwheels.org/project/nextbox-ui-plugin/
